.pre-commit-config.yaml

```diff
@@ -20,6 +20,11 @@ repos:
     rev: 0.15.0
     hooks:
       - id: check-github-actions
+        files: >
+          (?x)^(
+            action.(yml|yaml)|
+            \.github/actions/action.(yml|yaml)
+          )$
```

Comment:
temporary fix for https://github.com/python-jsonschema/check-jsonschema/pull/186